Locosys R1 RTK GPS receiver (#2055)
PX4/PX4-user_guide: en/gps_compass/rtk_gps_locosys_r1.md

# Locosys RTK GPS

> The [LOCOSYS Hawk R1](https://www.locosystech.com/en/product/hawk-r1.html) is a dual-frequency [RTK GPS module](../gps_compass/rtk_gps.md) receiver designed based on compatibility with Pixhawk from Locosys.

Reviewer: ```suggestion
The [LOCOSYS Hawk R1](https://www.locosystech.com/en/product/hawk-r1.html) is a dual-frequency [RTK GPS module](../gps_compass/rtk_gps.md) receiver designed for compatibility with Pixhawk.

```

The module can act as rover when installed on the aircraft. The receiver is capable of concurrently tracking all global civil navigation systems, including GPS, GLONASS, GALILEO, BEIDOU and QZSS. It acquires both L1 and L5 signals at the same time while providing the centimeter-level RTK positioning accuracy.

> The built-in lightweight helical antenna not only enhances RTK positioning stability, but also increases the flight time of the drone. The fast Time-To-First-Fix, RTK convergence, superior sensitivity, low power consumption make it a better choice for Pixhawk-based platform UAVs.

Reviewer: > but also increases the flight time of the drone. 

How? That sound unlikely.

Reviewer: OK, I assume this is because of the low power consumption.

```suggestion
The built-in lightweight helical antenna enhances RTK positioning stability.
The fast time-to-first-fix, RTK convergence, superior sensitivity, low power consumption make it a better choice for Pixhawk-based platform UAVs.

```

Main features include:
- Concurrent reception of L1 and L5 band signals
- Support GPS, GLONASS, BEIDOU, GALILEO, QZSS
- Capable of SBAS (WAAS, EGNOS, MSAS, GAGAN)
- Support 135-channel GNSS
- Fast TTFF at low signal level
- Free hybrid ephemeris prediction to achieve faster cold start
- Default 5Hz, up to 10 Hz update rate*
- Build-in super capacitor to reserve system data for rapid satellite acquisition

:::note
This module can be used with PX4 v1.13 or above.
:::

![LOCOSYS Hawk R1](../../assets/hardware/gps/locosys_hawk_a1/locosys_hawk_a1_gps.png)

## Where to Buy

* [LOCOSYS Hawk R1](https://www.locosystech.com/en/product/hawk-r1.html)

## Kit Contents

An RTK GPS kit includes:
- 1x GPS Module
- 1x Helix antenna
- 1x 6-pin JST-GH 

## Configuration

RTK setup and use on PX4 via *QGroundControl* is largely plug and play (see [RTK GPS](../advanced_features/rtk-gps.md) for more information).

> You need to connect your Hawk R1 to either GPS2 port on compatible Pixhawk boards or your preferred UART port on the boards without a seperate GPS2 port.

Reviewer: As I understand it you can use any port on any board.

```suggestion
Connect your Hawk R1 to the `GPS2` port on compatible Pixhawk boards (preferred, though you can use any other unused UART port).

```
